monai/networks/nets/segresnet_ds.py

```diff
@@ -508,8 +508,10 @@ def forward(  # type: ignore
 
         outputs: list[torch.Tensor] = []
         outputs_auto: list[torch.Tensor] = []
-        x_ = x.clone()
+        x_ = x
         if with_point:
+            if with_label:
+                x_ = x.clone()
```

Comment:
Hi @binliunls , could you explain the changes here? Why remove deepcopy when with_label is False?

Reply:
The input `x` is cloned here because there are two branches for inference. Therefore, in order to make the inference result correct, we need two copies of input to go through difference branch. However, when we only have one branch, there is no need to maintain this copy. So I move this copy logic here to avoid invalid copy.